assets/js/components/DateRangeSelector.js

```diff
@@ -100,13 +100,15 @@ export default function DateRangeSelector() {
 					'mdc-button--dropdown',
 					'googlesitekit-header__dropdown',
 					'googlesitekit-header__date-range-selector-menu',
+					'googlesitekit-border-radius-round--phone',
+					'googlesitekit-button-icon--phone',
 					{
 						'googlesitekit-header__date-range-selector-menu--has-unified-dashboard': unifiedDashboardEnabled,
 					}
 				) }
 				text
 				onClick={ handleMenu }
-				icon={ <DateRangeIcon width="18" height="20" /> }
+				icon={ <DateRangeIcon width="20" height="20" /> }
```

Comment:
The button for the date range selector has incorrect width since it has `padding-right: 8px` which sets the width to: `44px`. Can you review to make the button `36px` on small screens as per the AC?